.github/workflows/lint-docs-weekly.yml

```diff
@@ -10,6 +10,9 @@ permissions: {}
 jobs:
   markdown-link-check:
     name: Broken Links
+    strategy:
+      matrix:
+        branch: [ main, release-1.3, release-1.2 ]
     runs-on: ubuntu-latest
```

Comment:
Thx

One nit: please also add this to release-tasks.md ~ here: `Update `.github/workflows/scan.yml` to setup Trivy scanning for the currently supported branches.`